### PR TITLE
commenting linux-webkit project

### PR DESCRIPTION
**Pull request by skumar09**
PR Includes:

- Temporarily commenting Github Ubuntu - WebKit run,

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -59,12 +59,6 @@ const config = {
       },
     },
 
-    {
```
Comment (Dli3) [inline playwright.config.js:62]: If we're adding this back at a later time, can't we just comment this out for now?

**Review by Dli3 — APPROVED** [review]

File: run.sh [hunk 2]
```diff
@@ -84,7 +84,7 @@ if [[ -n  "$APPS" ]];then
           echo "*** npx playwright test --config=./configs/${conf_name}.config.js ${TAGS} --project=${app_name} ***"
           npx playwright test --config=./configs/${conf_name}.config.js ${TAGS} --project=${app_name}-chrome ${REPORTER}
           npx playwright test --config=./configs/${conf_name}.config.js ${TAGS} --project=${app_name}-firefox ${REPORTER} 
-          npx playwright test --config=./configs/${conf_name}.config.js ${TAGS} --project=${app_name}-webkit ${REPORTER}
+          # npx playwright test --config=./configs/${conf_name}.config.js ${TAGS} --project=${app_name}-webkit ${REPORTER}
```
Comment (amauch-adobe) [inline run.sh:87]: I might be jumping to conclusions, but if we comment this out, won't it also block other webkit runs on different OSs?

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
@skumar09 what is the issue here with the linux webkit? Also see my question below.

File: run.sh (hunk 2 above)
Comment (skumar09) [reply run.sh:87]: @amauch-adobe : 
- For `run-nala` label  `No` but for other labels `yes`, 
- I will revert it back in 2/3 days.

Comment (amauch-adobe) [reply run.sh:87]: Thank you for the clarification.

**Review by amauch-adobe — APPROVED** [review]
Approved.

**Pull request merged**